### ISSUE TITLE
Logarithm scale initialization

### ISSUE THREAD

**Issue by smileyenot983** [graphdeco-inria/gaussian-splatting#847]
A very interesting approach for scale initialization of gaussians. Bigger distance to closest points -> higher scale. Could you elaborate on why have you decided to take the logarithm of mean distances instead of using mean distance as is? Is it just to bound the values of scales?

**Comment by robofar** [comment]
So, what distCUDA2 computes actually? Is it for every point, it computes some distance which is mean of the distances of that point to its k closest points? Did they do this to set the scale of points that have neighbours that are further away, since that Gaussian will probably have to grow bigger to cover more area since there is no points near them?

**Comment by smileyenot983** [comment]
Yes, i suppose so. If there are less gaussians - then each single gaussian should cover larger area

**Comment by robofar** [comment]
@smileyenot983 
Do you know maybe what is role of scale activation? There are scales that are being optimized, but for rendering, to the renderer scales after activation are passed, instead of "plain" scales. Why is this? Why scale activation function is `exp`?
